tests/test_auto/test_optuna/test_config_sampler.py

```diff
@@ -0,0 +1,54 @@
+import os
+import time
+from random import SystemRandom
+
+import optuna
+import pytest
+from joblib import Parallel
+from joblib import delayed
+from optuna.storages import RDBStorage
+
+from etna.auto.optuna.config_sampler import ConfigSampler
+
+
+@pytest.fixture()
+def config_sampler():
+    return ConfigSampler(configs=[{"x": i} for i in range(10)])
+
+
+@pytest.fixture()
+def objective():
+    def objective(trial: optuna.trial.Trial):
+        rng = SystemRandom()
+        config = {**trial.relative_params, **trial.params}
+        time.sleep(10 * rng.random())
+        return (config["x"] - 2) ** 2
+
+    return objective
+
+
+@pytest.fixture()
+def sqlite_storage():
+    storage_name = f"{time.monotonic()}.db"
+    yield RDBStorage(f"sqlite:///{storage_name}")
+    os.unlink(storage_name)
+
+
+def test_config_sampler_one_thread(objective, config_sampler):
+
+    study = optuna.create_study(sampler=config_sampler)
+    study.optimize(objective, n_trials=100)
+    assert study.best_trial.user_attrs["pipeline"] == {"x": 2}
+    assert len(study.trials) == len(config_sampler.configs)
+
+
+@pytest.mark.skip
```

Comment:
But it can't be fixed)

Reply:
I suggested to make two tests out of this one, the first check that it chose the correct config(and it passes) and the second fails because of this problem